.travis.yml

```diff
@@ -1,7 +1,7 @@
 version: ~> 1.0 # needed for imports
 import: scala/scala-dev:travis/default.yml
 
-dist: xenial  # GPG stuff breaks on bionic; scala/scala-dev#764
+dist: noble
```

Comment:
The "language spec" stage below still uses `dist: focal`. I didn't change that because it seems more likely to break; I noted that the https://github.com/wkhtmltopdf it uses is marked "Archived" on GitHub. So just leaving this aside.